### PR TITLE
Remove references to single and multiple master clusters, per sdodson

### PR DESCRIPTION
**Pull request by mburke5678**
As suggested in email 6/21/18
sdodson: "I'd get rid of the references to single master, those went away after upgrading to 3.7."

### CODE REVIEW

**Comment by adellape** [comment]
Please hold on merging this til I get https://github.com/openshift/openshift-docs/pull/10230 merged.

**Comment by mburke5678** [comment]
@sdodson In a recent email to me, you suggested "I'd get rid of the references to single master, those went away after upgrading to 3.7." Can you or someone on your team take a look at this PR, an effort to get rid of those references? This was principally a find a remove/hide process and I want to make sure I am not removing something that should not be removed.

File: install/configuring_inventory_file.adoc [hunk 1]
```diff
@@ -59,6 +59,15 @@ xref:../architecture/core_concepts/containers_and_images.adoc#architecture-image
...
 
+[discrete]
+[[advanced-install-image-version-policy]]
+=== Image Version Policy
+
+Images require a version number policy in order to maintain updates. See
+the
+xref:../architecture/core_concepts/containers_and_images.adoc#architecture-images-tag-policy[Image
+Version Tag Policy] section in the Architecture Guide for more information.
+
```
Comment (sdodson) [inline install/configuring_inventory_file.adoc]: This change seems unrelated?

File: install/prerequisites.adoc [hunk 2]
```diff
@@ -321,10 +320,11 @@ include::install/prerequisites.adoc[tag=networkmanager]
 Similarly, if the `PEERDNS` parameter is set to `no` in the network script, for example, *_/etc/sysconfig/network-scripts/ifcfg-em1_*, then the dnsmasq files are not generated, and the Ansible install will fail. Ensure the `PEERDNS` setting is set to `yes`.
 ====
 
-The following is an example set of DNS records for the xref:index.adoc#single-master-multi-node[Single Master and Multiple Nodes] scenario:
+The following is an example set of DNS records:
 
 ----
 master    A   10.64.33.100
+master    A   10.64.33.103
```
Comment (sdodson) [inline install/prerequisites.adoc]: master1 master2?

**Review by sdodson — COMMENTED** [review]
In general looks fine to me, just two questions.

File: install/configuring_inventory_file.adoc (hunk 1 above)
Comment (mburke5678) [reply install/configuring_inventory_file.adoc]: @sdodson Thank you. Nice catch. Not sure how that got in there. This was added to the Advanced Install document in 2017, but changed to a note by @adellape when he reorganized that topic. Not sure how it got back in. Will remove!

**Comment by mburke5678** [comment]
@jiajliu Is this something your team can review for me?

**Comment by mburke5678** [comment]
@jiajliu Is this something your team can review for me?

**Comment by mburke5678** [comment]
@jiajliu Is this something your team can review for me?

**Comment by jianlinliu** [comment]
@sdodson One more confirm, do we really not support single master now??? This is a little surprise to me. As far as I know, many user (including user) is using single master for testing.

**Comment by sdodson** [comment]
@jianlinliu No, we do, we're just clarifying some wording and removing places that we referred to `atomic-openshift-master` as now all installs should use the service names that were used on multi-master environments.

**Comment by jianlinliu** [comment]
LGTM.

**Comment by mburke5678** [comment]
@openshift/team-documentation PTAL

File: install/prerequisites.adoc [hunk 3]
```diff
@@ -524,12 +524,7 @@ In the following table,
 *(L)* indicates the marked port is also used in _loopback mode_,
```
Comment (kalexand-rh) [inline install/prerequisites.adoc]: Is there any point where loopback mode, or the fact that you're using it, is important? (If not, this is a good time to remove both the note and the annotations in the table.)

File: architecture/infrastructure_components/kubernetes_infrastructure.adoc [hunk 4]
```diff
@@ -105,11 +104,6 @@ installation documentation] provides specific examples using the `native` HA met
 configuring HAProxy. You can also take the concepts and apply them towards your
 existing HA solutions using the `native` method instead of HAProxy.
 
-[NOTE]
```
Comment (kalexand-rh) [inline architecture/infrastructure_components/kubernetes_infrastructure.adoc]: Do we need to remove this note? Scott said that you can have single masters, so I think this is still relevant information.

File: install/prerequisites.adoc [hunk 5]
```diff
@@ -142,8 +142,7 @@ a master host in an {product-title} cluster of 2000 pods would be the minimum
 requirements of 2 CPU cores and 16 GB of RAM, plus 2 CPU cores and 3 GB of RAM,
 totaling 4 CPU cores and 19 GB of RAM.
 
-When planning an environment with multiple masters, a minimum of three etcd
-hosts and a load-balancer between the master hosts are required.
+A minimum of three etcd hosts and a load-balancer between the master hosts are required.
```
Comment (kalexand-rh) [inline install/prerequisites.adoc]: I might say "For high availability, a minimum..."

**Review by kalexand-rh — COMMENTED** [review]
Two questions, otherwise LGTM

File: install/prerequisites.adoc (hunk 3 above)
Comment (mburke5678) [reply install/prerequisites.adoc]: @sdodson Can you comment on @kalexand-rh comment? I am not familiar with loopback in this context. Is loopback mode required/default? I don't see anything on how to turn loopback on/off for the install. Is this note and the [L] annotations just informational? 

This note refers to Table 3 under Required Ports:
https://docs.openshift.com/container-platform/3.9/install_config/install/prerequisites.html#required-ports

File: architecture/infrastructure_components/kubernetes_infrastructure.adoc (hunk 4 above)
Comment (sdodson) [reply architecture/infrastructure_components/kubernetes_infrastructure.adoc]: Yes, this can be removed, you can scale up masters now. You probably couldn't when this was originally written.

File: install/prerequisites.adoc (hunk 3 above)
Comment (mburke5678) [reply install/prerequisites.adoc]: @sdodson Before we remove the info, do you have any thoughts on why the loopback notations were added? That PR, https://github.com/openshift/openshift-docs/pull/2215, comes from a customer issue, so I want to see if the information could be helpful.
https://access.redhat.com/support/cases/#/case/01544754 (In Japanese)

File: architecture/infrastructure_components/kubernetes_infrastructure.adoc (hunk 4 above)
Comment (sdodson) [reply architecture/infrastructure_components/kubernetes_infrastructure.adoc]: This should be fine for 3.7+ don't bother with anything older than 3.7.

File: install/prerequisites.adoc (hunk 3 above)
Comment (sdodson) [reply install/prerequisites.adoc]: I read the bug but wasn't quite sure of the purpose, I think the customer made an assumption that the ports are only necessary when operating multiple masters because the section is "Masters to Masters". I think that's an incorrect assumption to make, master-A -> master-A is still "Master to Master". I think as it is now it's a case of too much information being confusing and it should be removed.
@nak3 what do you think about removing these notes about loopback here?

Comment (nak3) [reply install/prerequisites.adoc]: Yes, I agree. It should be removed. I checked bz#1287949 and it looks like my first comment was wrong. I'm sorry :fearful:

**Comment by mburke5678** [comment]
@nak3 @sdodson @kalexand-rh I removed the loopback mode references and annotations as suggested. If no other objections or concerns, I will merge to 3.7+

**Comment by kalexand-rh** [comment]
SGTM @mburke5678. (FWIW, I vote to merge.)

**Comment by sdodson** [comment]
lets do this!

**Comment by ahardin-rh** [comment]
Just needs a squash! :eggplant:
